src/Leaves/BackgroundTaskViewBridge.js

```diff
@@ -36,7 +36,7 @@ bridge.prototype.start = function(){
     xmlhttp.onreadystatechange = function () {
 
         // if the ready state is now 4 we've already processed our entire output.
-        if (xmlhttp.completed || xmlhttp.readyState === 4) {
+        if (xmlhttp.completed) {
             document.body.classList.remove("event-processing");
```

Comment:
This shouldn't be needed due to the code below, but is shouldn't harm to be extra cautious. Happy to remove however!